script.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+if (localStorage.getItem("list of Books") === null) {
+  localStorage.setItem("list of Books", JSON.stringify([]));
+}
+
+// eslint-disable-next-line prefer-const
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.